compair/static/test/fixtures/instructor/default_fixture.js

```diff
@@ -120,53 +120,121 @@ storage.user_courses[student1.id] = [
 
 storage.course_assignments[course.id] = [];
 
-var assignment_finished = assignmentFactory.generateAssignment("1abcABC123-abcABC123_Z", instructor, [defaultCriterion, criterion3], {
+var assignment_with_feedback = assignmentFactory.generateAssignment("1abcABC123-abcABC123_Z", instructor, [defaultCriterion, criterion3], {
+    "name": "Assignment With Feedback",
+    "students_can_reply": true,
+    "available": true,
+    "compared": true,
+    "compare_period": false,
+    "after_comparing": true,
+    "answer_period": false,
+    "answer_start": Date.parse("Mon, 02 May 2016 23:59:59 -0000"),
+    "answer_end": Date.parse("Mon, 09 May 2016 23:59:59 -0000"),
+    "status": {
```

Comment:
endpoint dates should remain strings (using date objects here is only ok if they are converted back to strings)

Also in order to get upcoming assignment dates working properly, you will need to create some dynamic dates (I suggest using moment via npm)